File: .sdk-repos/orchestration-cluster-api-php/examples/audit_log.php
<?php

/**
 * Compilable usage examples for audit-log and form operations.
 */

declare(strict_types=1);

namespace Camunda\Orchestration\Examples;

use Camunda\Orchestration\CamundaClient;

// region SearchAuditLogs
/**
 * Search audit logs.
 */
function search_audit_logs(CamundaClient $client, ?\Camunda\Orchestration\Api\Model\AuditLogSearchQueryRequest $auditLogSearchQueryRequest = null): void
{
    $client->searchAuditLogs($auditLogSearchQueryRequest);
}
// endregion SearchAuditLogs

// region GetAuditLog
/**
 * Get audit log.
 */
function get_audit_log(CamundaClient $client, string $auditLogKey): void
{
    $client->getAuditLog($auditLogKey);
}
// endregion GetAuditLog

// region GetFormByKey
/**
 * Get form by key.
 */
function get_form_by_key(CamundaClient $client, string $formKey): void
{
    $client->getFormByKey($formKey);
}
// endregion GetFormByKey

File: .sdk-repos/orchestration-cluster-api-rust/examples/audit_log.rs
//! Compilable usage examples for audit log operations.
//!
//! Region-tagged snippets are surfaced in the Camunda REST API documentation via
//! `examples/operation-map.json`, and are type-checked by `cargo build --examples`.
#![allow(dead_code, unused_variables, unused_imports)]

use camunda_orchestration_sdk::apis::audit_log_api::GetAuditLogParams;
use camunda_orchestration_sdk::apis::audit_log_api::SearchAuditLogsParams;
use camunda_orchestration_sdk::models::AuditLogSearchQueryRequest;
use camunda_orchestration_sdk::CamundaClient;

// region GetAuditLog
async fn get_audit_log(audit_log_key: String) -> Result<(), Box<dyn std::error::Error>> {
    let client = CamundaClient::from_env()?;

    let result = client
        .get_audit_log(GetAuditLogParams { audit_log_key })
        .await?;
    println!("{}", result.entity_key);

    Ok(())
}
// endregion GetAuditLog

// region SearchAuditLogs
async fn search_audit_logs() -> Result<(), Box<dyn std::error::Error>> {
    let client = CamundaClient::from_env()?;

    let result = client
        .search_audit_logs(SearchAuditLogsParams {
            audit_log_search_query_request: Some(AuditLogSearchQueryRequest::default()),
        })
        .await?;
    for item in result.items {
        println!("{}", item.entity_key);
    }

    Ok(())
}
// endregion SearchAuditLogs

fn main() {
    // Examples above are compiled, not executed.
}
